docs/Job-reference.md

```diff
@@ -4255,6 +4290,40 @@ The arguments here are in order:
 * (String) the findbugs-files to parse
 * (boolean) use the findbugs rank for the priority, default to false
 
+### Phabricator Notifier
+
+```groovy
+job {
+    publishers {
+        phabricatorNotifier {
+            commentOnSuccess(boolean comment = true)                // defaults to false
+            commentWithConsoleLinkOnFailure(boolean comment = true) // defaults to false
+            coberturaReportFile(String coverageFile)                // defaults to false
+            enableUberalls(boolean enable = true)                   // defaults to false
```

Comment:
there is also a new option to drop text into a file (default `.phabricator-comment`) to post back:

https://github.com/uber/phabricator-jenkins-plugin#additional-comments

https://github.com/uber/phabricator-jenkins-plugin/blob/275cc0cf46c65305aba1e7c587967d7a8af59e20/src/main/resources/com/uber/jenkins/phabricator/PhabricatorNotifier/config.jelly#L20-L26